src/bp-notifications/bp-notifications-settings.php

```diff
@@ -0,0 +1,400 @@
+<?php
+/**
+ * Notifications Settings
+ *
+ * @package BuddyBoss\Notifications
+ * @since BuddyBoss [BBVERSION]
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Get the Notification settings sections.
+ *
+ * @return array
+ * @since BuddyBoss [BBVERSION]
+ */
+function bb_notification_get_settings_sections() {
+
+	$settings = array(
+		'bp_notifications'                   => array(
+			'page'              => 'notifications',
+			'title'             => __( 'On-screen Notifications', 'buddyboss' ),
+			'tutorial_callback' => 'bp_admin_on_screen_notification_setting_tutorial',
+			'notice'            => ( ! bb_enabled_legacy_email_preference() ) ? __( 'Members can manage which on-screen notifications they receive in their notification preferences by enabling or disabling the "Web" options.', 'buddyboss' ) : '',
+		),
+		'bp_notification_settings_automatic' => array(
+			'page'              => 'notifications',
+			'title'             => __( 'Notification Types', 'buddyboss' ),
+			'tutorial_callback' => 'bb_automatic_notifications_tutorial',
+			'notice'            => (
+				false === bb_enabled_legacy_email_preference() ?
+				sprintf(
+					wp_kses_post(
+							/* translators: Tutorial link. */
+						__( 'You can register your own notifications types by following the steps in %s. Once registered, they\'ll be configurable in the options above.', 'buddyboss' )
+					),
+					'<a href="' .
+					esc_url(
+						bp_get_admin_url(
+							add_query_arg(
+								array(
+									'page'    => 'bp-help',
+									'article' => 0,
+								),
+								'admin.php'
+							)
+						)
+					)
+					. '">' . esc_html__( 'this tutorial', 'buddyboss' ) . '</a>'
+				) : ''
+			),
+		),
+	);
+
+	return (array) apply_filters( 'bb_notification_get_settings_sections', $settings );
+}
+
+/**
+ * Link to Automatic Notification tutorial
+ *
+ * @since BuddyBoss [BBVERSION]
+ */
+function bb_automatic_notifications_tutorial() {
+	?>
+
+	<p>
+		<a class="button" href="
+		<?php
+		echo esc_url(
+			bp_get_admin_url(
+				add_query_arg(
+					array(
+						'page'    => 'bp-help',
+						'article' => 62829,
+					),
+					'admin.php'
+				)
+			)
+		);
+		?>
+		"><?php esc_html_e( 'View Tutorial', 'buddyboss' ); ?></a>
+	</p>
+
+	<?php
+}
+
+/**
+ * Get settings fields by section.
+ *
+ * @param string $section_id Section id.
+ *
+ * @return mixed False if section is invalid, array of fields otherwise.
+ * @since BuddyBoss [BBVERSION]
+ */
```

Comment:
/**
 * Get settings fields by section.
 * 
 * @since BuddyBoss [BBVERSION]
 * 
 * @param string $section_id Section id.
 *
 * @return mixed False if section is invalid, array of fields otherwise.
 */